napari/components/camera.py

```diff
@@ -59,14 +58,12 @@ class Camera(EventedModel):
         sets of Euler angles may lead to the same view.
     perspective : float
         Perspective (aka "field of view" in vispy) of the camera (if 3D).
-    interactive : bool
-        If the camera mouse pan/zoom is enabled or not.
-        This attribute is deprecated since 0.5.0 and should not be used.
-        Use the mouse_pan and mouse_zoom attributes instead.
     mouse_pan : bool
         If the camera interactive panning with the mouse is enabled or not.
+        Camera interactive panning with the mouse was removed in napari 0.6.0
```

Comment:
```suggestion
```
I don't think this is correct. The old way was `interactive` which controlled both zoom and pan. That was split into two. So panning with mouse is still a thing.